pkg/services/ngalert/api/api_prometheus.go

```diff
@@ -63,6 +65,38 @@ func (srv PrometheusSrv) RouteGetAlertStatuses(c *models.ReqContext) response.Re
 	return response.JSON(http.StatusOK, alertResponse)
 }
 
+func formatValues(alertState *state.State) string {
+	var fv string
+	values := alertState.GetLastEvaluationValuesForCondition()
+
+	switch len(values) {
+	case 0:
+		fv = alertState.LastEvaluationString
+	case 1:
+		vs := make([]string, 0, len(values))
+		for _, v := range values {
+			vs = append(vs, strconv.FormatFloat(v, 'e', -1, 64))
+		}
+
+		fv = vs[0]
+	default:
+		vs := make([]string, 0, len(values))
+
+		for k, v := range values {
+			vs = append(vs, fmt.Sprintf("%s: %s", k, strconv.FormatFloat(v, 'e', -1, 64)))
+		}
+
+		// Ensure we have a consistent ordering after formatting e.g. A0, A1, A3, etc.
+		sort.Slice(vs, func(i, j int) bool {
```

Comment:
The sort here is not a natural sort. I.e. the order of `["A2", "A1", "A10"]` will be `["A1", "A10", "A2"]`.

Reply:
I chose to do `strconv.Atoi(vs[i])` as it's the simplest thing I can do without going down a rabbithole and the position is being added by me. The bad thing here is that refIDs can be literally anything IIRC.

Reply:
After going down a rabbit hole here, I ended up accepting the fact that it'll not be a natural sort. 

If somehow this becomes a problem down the line - we can easily spend some time providing a logical order similar to what the UI gives us. For now, the use of `sort.Strings` will have to do.